File: app/src/main/java/com/example/msdc/api/PersonResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class PersonResult {
    @SerializedName("id")
    int id;

    @SerializedName("profile_path")
    String posterPath;

    @SerializedName("name")
    String name;

    public int getId() {
        return id;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/example/msdc/api/PersonResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class PersonResult (
    @field:SerializedName("id") val id: Int? = 0,
    @field:SerializedName("profile_path") val posterPath: String? = null,
    @field:SerializedName("name") val name: String? = null
) : Parcelable